src/praisonai-bot/praisonai_bot/gateway/server.py

```
@@ -6371,60 +6383,82 @@ def _create_bot(
+        for param, env_key in extras.items():
+            env_val = os.environ.get(env_key, "")
+            if env_val:
+                init_kwargs[param] = env_val
+        for key, value in ch_cfg.items():
+            if key in ("platform", "token"):
+                continue
+            init_kwargs[key] = value
+
+        try:
+            return adapter_cls(**init_kwargs)
+        except Exception as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)